otri/downloader/gme_downloader.py

```diff
@@ -113,10 +114,10 @@ def __prepare_data(dict_data: dict) -> dict:
             elif(isinstance(xs_element, list)):
                 req_types = [dict['@name'] for dict in xs_element]
             else:
-                print("req_type is non of the above: ", type(xs_element))
+                log.e("req_type is none of the above: {}".format(type(xs_element)))
                 return False
         except TypeError as error:
-            print("Unable to retrieve req_type, ", error)
+            log.e("Unable to retrieve req_type, ", error)
```

Comment:
Codacy found an issue: [Too many positional arguments for function call](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5768570)